examples/logconcave/CMakeLists.txt

```diff
@@ -8,13 +8,9 @@
 project( VolEsti )
 
 
-option(DISABLE_NLP_ORACLES "Disable non-linear oracles (used in collocation)" ON)
-option(BUILTIN_EIGEN "Use eigen from ../external" OFF)
-option(USE_MKL "Use MKL library to build eigen" ON)
-
 CMAKE_MINIMUM_REQUIRED(VERSION 2.4.5)
 
-set(MKLROOT $ENV{HOME}/intel/mkl)
+set(MKLROOT /opt/intel/oneapi/mkl/2022.1.0)
```

Comment:
should it be mkl/latest instead of the specific version?

Reply:
Yup,both works but latest is better.